nionswift_plugin/usim/ScanDevice.py

```diff
@@ -46,6 +49,84 @@ def __init__(self, frame_number: int, channels: typing.List[Channel], frame_para
         self.scan_data: typing.Optional[typing.List[_NDArray]] = None
 
 
+class ScanBoxSimulator:
```

Comment:
Can you add a comment to explain this class purpose? It would be good to include a high level explanation of the blanker simulator here too.